utils/src/lib.rs

```diff
@@ -729,7 +730,21 @@ pub fn abi_to_solidity(contract_abi: &Abi, mut contract_name: &str) -> Result<St
                 contract_name, events, structs, functions
             ),
         }
-    })
+    };
+
+    // Format generated solidity
+    let parsed = parse(&sol).map_err(|diags| {
+        eyre::eyre!(
+            "Failed to parse Solidity code for generated code. Leaving source unchanged.\nDebug info: {:?}",
+            diags
+        )
+    })?;
+
+    let mut output = String::new();
+    format(&mut output, parsed, FormatterConfig::default())
+        .map_err(|err| eyre::eyre!(err.to_string()))?;
```

Comment:
lgtm, but looks like we can add a simple function directly in `fmt` that formats a single file?